jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -13,6 +13,10 @@
 #Required only for PoDLE calculation:
 N = 115792089237316195423570985008687907852837564279074904382605163141518161494337
 
+BTC_P2PK_VBYTE = {"mainnet": b'\x00', "testnet": b'\x6f'}
+BTC_P2SH_VBYTE = {"mainnet": b'\x05', "testnet": b'\xc4'}
+PODLE_COMMIT_FILE = None
```

Comment:
I think `PODLE_COMMIT_FILE` can be removed. It's only used in `jmclient/podle.py`, which defines the variable itself.

Reply:
Good catch! Done.